templates/consonant/consonant.js

```diff
@@ -726,6 +726,11 @@ export function externalLinks(selector) {
     if (linkValue.includes('//') && !linkValue.includes('pages.adobe')) {
       linkItem.setAttribute('target', '_blank');
     }
+    if (window.pages.product && !linkValue.includes(window.pages.product)) {
```

Comment:
```suggestion
    } else if (window.pages.product && !linkValue.includes(window.pages.product)) {
```
This is my only suggestion since they are same purpose that detecting whether it is external or internal link.
So it is better for readability and reduce checking condition if it is not pages.adobe already.